README.md

```diff
@@ -1,34 +1,38 @@
-Workflows for Mailbox detection
+Workflows for Mailbox Detection
 -----------------------------------
+This workflow illustrates the integration of UiPath and Konduit-Serving with the example of mailbox detection. 
+This workflow able to detect the location of each mailbox and whether it's empty or not.  
+![MailBox Result](img/mailboxbeforeafter.PNG "MailBox Result")
 
 File Descriptions
 ----------------------
-ScoreImage.xaml: For use after the server is started. This allows you to send an image specified as a file path to 
-the server for detection. This is meant to be more of an example that should be integrated in to a larger workflow.
+**Main.xaml**: The main entry point. This contains the whole workflow for starting a server.
 
-StartServer.xaml: The main entry point. This contains the whole workflow for starting a server.
+**ScoreImage.xaml**: For use after the server is started. This allows you to send an image specified as a file path to 
+the server for detection. This is meant to be more of an example that should be integrated in to a larger workflow.
 
-DownloadAndExtract.xaml automatically downloads and extracts a server where the
+**DownloadAndExtract.xaml**: This automatically downloads and extracts a server where the
 mail box detection model will run
 
-ReplaceJsonWithPath.xaml: Handles configuring the server configuration (a json file) to point to your current directory
+**ConfigureServer.xaml**: Handles configuring the server configuration (a json file) to point to your current directory
 where you are running the workflow. This is necessary to start the server it (happens automatically)
 
 Intended usage
 ------------------------------------
-
-Run StartServer.xaml first. This will download a server distribution from the releases
+### Step 1: 
+Run Main.xaml first. This will download a server distribution from the releases
 of this repository containing everything needed to run the model.
```

Comment:
shall we add a short description that the estimated download time will be roughly 30min and in between there are nth gonna happen, hence please be patient?

Reply:
That is a good input. I was meaning to add that. Let me add that in.